src/zenml/integrations/evidently/visualizers/evidently_visualizer.py

```diff
@@ -51,7 +52,7 @@ def generate_facet(self, html_: str) -> None:
         Args:
             html_: HTML represented as a string.
         """
-        if self.running_in_notebook():
+        if Environment.in_jupyter_notebook():
```

Comment:
Probably makes sense to add `or Environment.in_colab` here? If that isn't needed then I wouldn't call this `in_jupyter_notebook` rather just `in_notebook`

Reply:
The visualizations worked in colab before right? Then it probably works in both cases, I will try later an either update the name or add the `or Environment.in_colab`

Reply:
Works for both, so I renamed it to `in_notebook`